### DISCUSSION TITLE
Should use Text with truncate and display tooltip when resizing columns

### DISCUSSION THREAD

**Discussion by n3x7j2** [KevinVandy/mantine-react-table#11] [General]
As you can see when resizing columns it looks like 
![image](https://user-images.githubusercontent.com/5088270/219347119-78b1efc8-28f7-42a8-a4fd-53cac1be992e.png)
 I suggest use truncate text for header, body, and footer with a tooltip

**Comment by KevinVandy** [comment]
You can do that yourself with the column Cell option to override the default cell render
